test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -200,6 +219,7 @@ describe('33acrossBidAdapter:', function () {
 
   afterEach(function() {
     sandbox.restore();
+    spec.adapterState.uniqueSiteIds = [];
```

Comment:
This is what I mean by the unit tests being stateful. I am forced to reset states between tests for each item in adapter spec.

Reply:
is it possible to clear `adapterState.uniqueSiteIds` in the `getUserSyncs()` function? so we do not need to export it and reset it

Reply:
Done.